redis.conf

```diff
@@ -150,6 +150,11 @@ tcp-keepalive 300
 #
 # tls-cert-file redis.crt 
 # tls-key-file redis.key
+#
+# If the key file is encrypted using a passphrase, it can be included here
+# as well.
+#
+# tls-key-file-pass secret
```

Comment:
do we wanna mention that it would be unsafe to pass this in the config file and advise stdin?

Reply:
I think `stdin` may add false sense of security, because it's still accessible through `CONFIG GET` and will get written by `CONFIG REWRITE`.

We should address this separately with an option to secure config parameters, but I see it as a separate issue as [described here](https://github.com/redis/redis/issues/8055#issuecomment-731266215).